src/data/activityData.json

```diff
@@ -1,251 +1,245 @@
+  },
+  {
+    "activitiesId": "ACT008",
+    "activityName": "Rhodo Mansion Meesappulimala",
+    "activityRouteId": "R4",
+    "bannerImg": "https://cdn.hellomunnar.in/activities/Rhodo_Mansion_Meesappulimala.jpg",
+    "activityDifficulty": "adventure",
+    "activityPrice": 3000,
+    "activityChallangesId": ["CH-3", "CH-4", "CH-5"],
+    "activitycoordinateslat": 10.0981631951696,
+    "activitycoordinateslon": 77.2032882028,
+    "bookingLink": "https://munnar.kfdcecotourism.com/",
+    "bookingContactName": "KFDC Ecotourism MUNNAR Mattupatty Road, Munnar",
+    "bookingContactNumber": "91 82898 21408",
+    "activityDescription": "Rhodo Mansion is an amazing accommodation facility under Kerala Forest Development\r\nCorporation(KFDC) exclusively for those who want to be a little bit adventurous by having a\r\ntrekking trail via Rhodo Valley. Located at a high altitude in the misty hilltop, Rhodo Mansion\r\nis a beautifully designed luxury option for those who love the wilderness and scenic beauty\r\nof Meesapulimala. The views from this mansion are truly amazing and stunning. Night stay\r\nat Rhodo mansion by enjoying a campfire and the delicious local cuisine is one of the\r\ngreatest experiences one can have."
+  },
+  {
+    "activitiesId": "ACT009",
+    "activityName": "Jungle Camp Pampadum Shola National Park",
+    "activityRouteId": "R1",
+    "bannerImg": "https://cdn.hellomunnar.in/activities/Jungle_Camp_Pampadum_Shola_National_Park.jpg",
+    "activityDifficulty": "adventure",
+    "activityChallangesId": ["CH-4"],
+    "activitycoordinateslat": 10.12665808,
+    "activitycoordinateslon": 77.25823467,
+    "bookingLink": "https://eravikulamnationalpark.in/landing/index.php",
+    "bookingContactName": "Wildlife Warden Munnar, Wildlife division, Munnar",
+    "bookingContactNumber": 8547603199,
+    "activityDescription": "Despite being the smallest national park in Kerala, the splendor of the shola forests in\r\nPampadum Shola National Park will mesmerize everyone’s mind. The Log House provides a\r\njungle camp package situated in the heart of shola forests. It costs 3000 rupees for 2 people."
+  },
+  {
+    "activitiesId": "ACT010",
+    "activityName": "Eravikulam National Park Jungle Camp",
+    "activityRouteId": "R1",
+    "bannerImg": "https://cdn.hellomunnar.in/activities/Eravikulam_National_Park_Jungle_Camp.jpg",
+    "activityOpenTime": "3 : 00 PM",
+    "activityCloseTime": "10.00 AM",
```

Comment:
There seems to be some descrepancy here on the closetime. Shouldn't it be `10 : 00`

Reply:
@athul yes, the data team is on it. Merge it for now.

Reply:
Wokay